# Specifies 7-DOF joint position to which Sawyer should move.
#
# j#pos: position of the #th joint
float32 j0pos
float32 j1pos
float32 j2pos
float32 j3pos
float32 j4pos
float32 j5pos
float32 j6pos
# Alternatively, you can access a preset joint configuration by referencing its name.
string name
# Optional: Fraction of fastest speed the arm can move
float32 speed_ratio
# Optional: Audio file delay
bool wait
# Optional: Failure plan if trajectory cannot be executed.
string fail_plan
---
bool success
---
bool feedback